packages/fxa-settings/src/models/_mocks.tsx

```diff
@@ -165,6 +171,42 @@ export function renderWithRouter(
   };
 }
 
+const settingsFtlPath = path.resolve(
+    __dirname,
+    '..',
+    '../public/locales/en-US/settings.ftl'
+);
+fetchMock.get('/locales/en-US/settings.ftl', fs.readFileSync(settingsFtlPath, 'utf-8'));
+fetchMock.get('/locales/en-GB/settings.ftl', fs.readFileSync(settingsFtlPath, 'utf-8'));
+fetchMock.get('/locales/en/settings.ftl', fs.readFileSync(settingsFtlPath, 'utf-8'));
+export async function renderWithRouterAndLocalization(
+    ui: any,
+    { route = '/', history = createHistory(createMemorySource(route)) } = {}
+) {
+  sinon.spy(AppLocalizationProvider.prototype, 'render');
+  const renderResult = {
+    ...render(
+        <AppLocalizationProvider
+            bundles={['settings']}
+            userLocales={['en-US']}
+        >
+          <LocationProvider {...{history}}>{ui}</LocationProvider>
+        </AppLocalizationProvider>)
+    ,
+    history,
+  };
+  await waitUntil(
+      () => {
+        // @ts-ignore
+        return AppLocalizationProvider.prototype.render.callCount === 2
```

Comment:
`(AppLocalizationProvider.prototype.render as sinon.SinonSpy).callCount` instead of `@ts-ignore`?